File: app/src/main/java/vinova/henry/com/hotfilm/server/ITrailerService.java
package vinova.henry.com.hotfilm.server;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
import vinova.henry.com.hotfilm.models.TrailerResult;

public interface ITrailerService {
    @GET("videos?")
    Call<TrailerResult> getTrailerData(@Query("api_key") String api_key);
}

File: app/src/main/java/vinova/henry/com/hotfilm/server/ITrailerService.kt
package vinova.henry.com.hotfilm.server

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query
import vinova.henry.com.hotfilm.models.TrailerResult

interface ITrailerService {
    @GET("videos?")
    fun getTrailerData(@Query("api_key") api_key: String): Call<TrailerResult>
}
